### ISSUE TITLE
6.16.2 version  GLOB_BRACE Constant error in Alpine system

### ISSUE THREAD

**Issue by dh9527** [spatie/laravel-backup#1369]
> **warning:  Backup failed because: Use of undefined constant GLOB_BRACE - assumed 'GLOB_BRACE' (this will throw an Error in a future version of PHP).**

file path : https://github.com/spatie/laravel-backup/blob/6.16.2/src/Tasks/Backup/FileSelection.php

``` php
protected function sanitize($paths): Collection
    {
        return collect($paths)
            ->reject(function ($path) {
                return $path === '';
            })
            ->flatMap(function ($path) {
                return glob(str_replace('*', '{.[!.],}*', $path), GLOB_BRACE);
            })
            ->map(function ($path) {
                return realpath($path);
            })
            ->reject(function ($path) {
                return $path === false;
            });
    }
```

**Comment by dh9527** [comment]
Backup failed because: Use of undefined constant GLOB_BRACE - assumed 'GLOB_BRACE' (this will throw an Error in a future version of PHP).

**Comment by erikn69** [comment]
Please try `defined('GLOB_BRACE') ? GLOB_BRACE : 0`

```php
->flatMap(function ($path) {
    return glob(str_replace('*', '{.[!.],}*', $path), defined('GLOB_BRACE') ? GLOB_BRACE : 0);
})
```
I don't use alpine, if that works i make a PR

**Comment by dh9527** [comment]
> Please try `defined('GLOB_BRACE') ? GLOB_BRACE : 0`
> 
> ```
> ->flatMap(function ($path) {
>     return defined('GLOB_BRACE') ? glob(str_replace('*', '{.[!.],}*', $path), GLOB_BRACE) :  glob($path);
> })
> ```
> 
> I don't use alpine, if that works i make a PR

Thank you. I think this should work normally, but I have updated to version 7.6.4 and haven't tested it.

**Comment by freekmurze** [comment]
This will be handled in an open PR.

**Comment by sinbino** [comment]
This problem has not been fixed in 6.16.3.
